kong/plugins/prometheus/exporter.lua

```diff
@@ -30,6 +31,12 @@ local function init()
   metrics.db_reachable = prometheus:gauge("datastore_reachable",
                                           "Datastore reachable from Kong, " ..
                                           "0 is unreachable")
+  metrics.upstream_target_health = prometheus:gauge("upstream_target_health",
+                                          "Exposes the state of the target of an upstream. " ..
```

Comment:
Let's change this to `Health status of targets of upstream`. Please update this in README.md as well.

Reply:
done!